opennms-model/src/main/java/org/opennms/netmgt/model/OnmsApplication.java

```diff
@@ -116,35 +107,47 @@ public void setName(String name) {
     @XmlElementWrapper(name="monitoredServices")
     @JsonBackReference
     public Set<OnmsMonitoredService> getMonitoredServices() {
-        return m_monitoredServices;
+        return monitoredServices;
     }
 
-    /**
-     * <p>setMonitoredServices</p>
-     *
-     * @param services a {@link java.util.Set} object.
-     * @since 1.8.1
-     */
     public void setMonitoredServices(Set<OnmsMonitoredService> services) {
-        m_monitoredServices = services;
+        monitoredServices = services;
     }
 
-    /**
-     * <p>addMonitoredService</p>
-     *
-     * @param service a {@link org.opennms.netmgt.model.OnmsMonitoredService} object.
-     * @since 1.8.1
-     */
     public void addMonitoredService(OnmsMonitoredService service) {
         getMonitoredServices().add(service);
     }
 
-    /**
-     * <p>compareTo</p>
-     *
-     * @param o a {@link org.opennms.netmgt.model.OnmsApplication} object.
-     * @return a int.
-     */
+
+    @ManyToMany( cascade={CascadeType.PERSIST, CascadeType.MERGE})
+    @JoinTable(name="application_perspective_location_map",
+            joinColumns=@JoinColumn(name="appId", referencedColumnName = "id"),
+            inverseJoinColumns=@JoinColumn(name="monitoringLocationId", referencedColumnName = "id"))
+    @XmlIDREF
+    @XmlElement(name="perspectiveLocationId")
+    @XmlElementWrapper(name="perspectiveLocations")
+    public Set<OnmsMonitoringLocation> getPerspectiveLocations() {
+        return this.perspectiveLocations;
+    }
+
+    public void setPerspectiveLocations(Set<OnmsMonitoringLocation> perspectiveLocations) {
+        this.perspectiveLocations = perspectiveLocations;
+    }
+
+    public void addPerspectiveLocation(OnmsMonitoringLocation perspectiveLocation) {
+        getPerspectiveLocations().add(perspectiveLocation);
+    }
+
+    /** Can be null. */
+    @Column(name = "polling_package", nullable=true)
```

Comment:
The column's name in the schema is "pollingpackage" not "polling_package".

Reply:
good catch! I changed the name but forgot here. fixed.